### PR TITLE
Issue 131

### LINKED ISSUE #131: Update output format
- when Fa is null 
  - include fa_error with message
  - ensure sms, sds, sdcs are null

- when Fv is null 
  - include fv_error with message
  - ensure sm1, sd1, sdc1 are null

- When either Fa or Fv is null
  - ensure sdc, sdSpectrum, smSpectrum are null

### LINKED ISSUE #132: Update Calc to handle null
When Fa or Fv are null make sure downstream calc work.

### PR DESCRIPTION
**Pull request by jdbrown-USGS**
fixes #131 
fixes #132

### CODE REVIEW

File: src/lib/asce7_16-factory.js [hunk 1]
```diff
@@ -152,6 +152,10 @@ var ASCE7_16Factory = function (options) {
    *     The seismic design value
    */
   _this.computeDesignValue = function (siteModifiedValue) {
```
Comment (ehunter-usgs) [inline src/lib/asce7_16-factory.js]: Add tests for `null` check
